metadata-service/factories/src/main/java/com/linkedin/gms/factory/auth/DataHubTokenServiceFactory.java

```diff
@@ -16,21 +19,35 @@ public class DataHubTokenServiceFactory {
   @Value("${authentication.tokenService.signingKey:}")
   private String signingKey;
 
+  @Value("${authentication.tokenService.saltingKey:}")
+  private String saltingKey;
+
   @Value("${elasticsearch.tokenService.signingAlgorithm:HS256}")
   private String signingAlgorithm;
 
   @Value("${elasticsearch.tokenService.issuer:datahub-metadata-service}")
   private String issuer;
 
+  /**
+   * +  @Inject
+   * +  @Named("entityService")
+   * +  private EntityService _entityService;
+   * +
+   */
+  @Autowired
+  @Qualifier("entityService")
+  private EntityService _entityService;
```

Comment:
See how other fields don't use underscore in their name? 

Let's keep it consistent so its less messy.